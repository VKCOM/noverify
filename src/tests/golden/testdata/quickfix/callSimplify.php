<?php

$array = ['a' => 1, 'b' => 2, 'c' => 3];

if (in_array('z', array_keys($array))) { // 2
    echo "contains z key\n";
}

function f1($k, $xs) {
    return in_array($k . '_id', array_keys($xs['data'][0])); // 1
}

function f2($k, $xs) {
    return in_array($k . '_id', $xs['data'][0]);
}
