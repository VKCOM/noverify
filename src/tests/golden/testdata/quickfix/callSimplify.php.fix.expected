<?php

$array = ['a' => 1, 'b' => 2, 'c' => 3];

if (array_key_exists('z', $array)) { // 2
    echo "contains z key\n";
}

function f1($k, $xs) {
    return array_key_exists($k . '_id', $xs['data'][0]); // 1
}

function f2($k, $xs) {
    return in_array($k . '_id', $xs['data'][0]);
}
